daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -458,53 +413,94 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments)
   object test extends scallop.Subcommand("test") {
-    banner("""|Usage: daffodil test <tdmlfile> [testname...]
+    banner("""|Usage: daffodil test [-l] [-r] [-i] <tdmlfile> [testname...]
               |
               |List or execute tests in a TDML file
               |
               |Test Options:""".stripMargin)
 
-    descr("list or execute TDML tests")
+    descr("List or execute TDML tests")
     helpWidth(76)
 
-    val list = opt[Boolean](descr = "show names and descriptions instead of running test cases.")
-    val regex = opt[Boolean](descr = "treat <names> as regular expressions.")
-    val tdmlfile = trailArg[String](required = true, descr = "test data markup language (TDML) file.")
-    val names = trailArg[List[String]](required = false, descr = "name of test case(s) in tdml file. If not given, all tests in tdmlfile are run.")
-    val info = tally(descr = "increment test result information output level, one level for each -i")
+    val info = tally(descr = "Increment test result information output level, one level for each -i")
+    val list = opt[Boolean](descr = "Show names and descriptions instead of running test cases")
+    val regex = opt[Boolean](descr = "Treat <testname...> as regular expressions")
+    val tdmlfile = trailArg[String](required = true, descr = "Test Data Markup Language (TDML) file")
+    val names = trailArg[List[String]](required = false, descr = "Name(s) of test cases in tdmlfile. If not given, all tests in tdmlfile are run.")
+  }
+
+  // Performance Subcommand Options
+  object performance extends scallop.Subcommand("performance") {
+    banner("""|Usage: daffodil performance (-s <schema> [-r [{<namespace>}]<root>] [-p <path>] |
+              |                             -P <parser>)
+              |                            [-D[{<namespace>}]<variable>=<value>...]
+              |                            [-u] [-V [mode]] [-N <number>] [-t <threads>]
+              |                            [-I <infoset_type>] [-c <file>]
+              |                            <infile>
+              |
+              |Run a performance test, using either a DFDL schema or a saved parser
+              |
+              |Performance Options:""".stripMargin)
+
+    descr("Run performance test")
+    helpWidth(76)
+
+    val config = opt[File](short = 'c', argName = "file", descr = "Path to XML file containing configuration items")
+    val vars = props[String]('D', keyName = "variable", valueName = "value", descr = "Variables to be used when parsing or unparsing. An optional namespace may be provided.")
+    val infosetType = opt[InfosetType.Type](short = 'I', argName = "infoset_type", descr = "Infoset type to output or unparse. Must be one of 'xml', 'scala-xml', 'json', 'jdom', 'w3cdom', 'sax', or 'null'.", default = Some(InfosetType.XML))
+    val number = opt[Int](short = 'N', argName = "number", default = Some(1), descr = "The total number of files to process. Defaults to 1.")
```

Comment:
I just noticed that sometimes we include a "Defaults to .." message, but sometimes we don't. For example, infoseType defaults to "xml".  Should we be consistent, and any option that has a default value, we should mention what it defaults to in the help string?